nautobot_golden_config/api/serializers.py

```diff
@@ -68,6 +69,8 @@ class GoldenConfigSettingSerializer(TaggedObjectSerializer, CustomFieldModelSeri
     url = serializers.HyperlinkedIdentityField(
         view_name="plugins-api:nautobot_golden_config-api:goldenconfigsetting-detail"
     )
+    scope = serializers.JSONField(read_only=True)
```

Comment:
Can we leverage the `to_internal_value` method on the write? We can auto-generate a dynamic group on this.

Reply:
Doing a setter on the model instead.